<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="cbx_sort">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Фамилии</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Имени</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Отчеству</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_hint">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Искать по</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_input">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Искать</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>721</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_changeinfo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_getinfo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить информацию</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_info">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
